packages/fleather/lib/src/rendering/editor.dart

```diff
@@ -861,6 +861,7 @@ class RenderEditor extends RenderEditableContainerBox
       _floatingCursorRect = null;
       _cursorController.setFloatingCursorTextPosition(null);
     }
+    markNeedsPaint();
```

Comment:
I've experiment with and without this line and saw no changes. Can you explain why it was added? And maybe add a test for that so we don't forget in the future?

Reply:
The floating cursor stay until the next tap or edit action
In the video, notice the floating cursor that remains after releasing the floating cursor

https://github.com/user-attachments/assets/b2d6ebbe-3df2-42f0-9b60-2ede0d76eb81

Reply:
Then I guess no need for a test. It would be really hard to test it I guess.